web_responsive/static/src/css/web_responsive.scss

```
@@ -131,16 +131,46 @@ $chatter_zone_width: 35%;
         }
     }
 }
+.o_main_navbar {
+    color: color-yiq($o-brand-odoo);
 
+    > ul > li > a,
+    > ul > li > label {
+        color: color-yiq($o-brand-odoo);
+    }
+    .dropdown-menu.show {
+        max-height: calc(100vh - #{$o-navbar-height});
+    }
+}
 // Iconized full screen apps menu
 .o_menu_apps {
     .search-input:focus {
         border-color: $o-brand-primary;
     }
 
+    a.full {
+        width: $o-navbar-height;
+        text-align: center;
+    }
+
     .dropdown-menu.show {
+        visibility: visible;
+        opacity: 1;
```

No need to handle the `visibility` property, but we need to animate `opacity`:

```suggestion
        transition: opacity 100ms ease;
        opacity: 1;
```

(I used 100ms because it looks better IMHO, but you can use 50ms if you prefer. I'm no designer.)